### PR TITLE
Changed some doc links to use intersphinx. 

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

**Comment by felixxm** [comment]
I'm not sure, tutorial fits better to How-to.

File: docs/howto/windows.txt [hunk 1]
```diff
@@ -57,8 +57,8 @@ It is best practice to provide a dedicated environment for each Django project
 you create. There are many options to manage environments and packages within
 the Python ecosystem, some of which are recommended in the `Python
 documentation <https://packaging.python.org/guides/tool-recommendations/>`_.
-Python itself comes with `venv`_ for managing environments which we will use
-for this guide.
+Python itself comes with :mod:`venv` for managing environments which we will
+use for this guide.
```
Comment (adamchainz) [inline docs/howto/windows.txt:61]:
```suggestion
Python itself comes with :doc:`python:tutorial/venv` for managing environments
which we will use for this guide.
```

**Comment by jdufresne** [comment]
Thanks @felixxm. I totally overlooked that it wasn't linking to the module docs. I've incorporated the suggestion from @adamchainz. When doing so, I noticed another link that could use intersphinx so I updated that one too.

File: docs/howto/windows.txt (hunk 1 above)
Comment (jdufresne) [reply docs/howto/windows.txt]: I've added this but also overrode the text. Sphinx's default uses the doc's title which is "Virtual Environments and Packages". The override is the original "venv".

**Comment by felixxm** [comment]
@jdufresne Thanks :+1: I would add `python:` e.g. ```doc:`venv <python:tutorial/venv>` ``` to make sure that links don't point to other docs.

**Comment by adamchainz** [comment]
+1 to `python:` - explicit better than implicit 😉

**Comment by jdufresne** [comment]
:+1: I added the `python:` prefix.

**Pull request merged**